apps/judicial-system/backend/src/app/modules/case/internalCase.service.ts

```diff
@@ -587,6 +575,15 @@ export class InternalCaseService {
             this.config.archiveEncryptionKey,
             { iv: CryptoJS.enc.Hex.parse(uuidFactory()) },
           ).toString(),
+          // To decrypt:
```

Comment:
Is this meant to be a part of this PR?

Reply:
Yes, every time we need to decrypt the arvhived information we have had a hard time figuring out how.